src/commands/commandui.cpp

```diff
@@ -172,6 +168,15 @@ int CommandUI::run(QStringList& tokens) {
     // If there is another instance, the execution terminates here.
     if (!EventListener::checkOtherInstances(
             I18nStrings::instance()->t(I18nStrings::ProductName))) {
+      // If we are given URL parameters, send them to the UI socket and exit.
+      for (const QString& value : tokens) {
```

Comment:
Is the "ui" parameter in `mozillavpn ui %u` being checked for and verified?

Reply:
The handling of this command parameter takes place in CommandLineParser::parse()